src/services/api/exposures.ts

```diff
@@ -113,7 +113,7 @@ export const uploadExposureKeys = async (uploadToken: string, symptomDate: strin
       key: exposure.keyData,
       rollingStartNumber: exposure.rollingStartNumber,
       rollingPeriod: exposure.rollingPeriod,
-      transmissionRiskLevel: exposure.transmissionRiskLevel
+      transmissionRisk: exposure.transmissionRiskLevel
```

Comment:
Does the module @1.1.24 still accept and support `transmissionRiskLevel` if passed in? @colmharte

Reply:
It does but APHL expects it to be named `transmissionRisk`